src/Stubs/common/Component/HttpKernel/Kernel.stubphp

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Symfony\Component\HttpKernel;
+
+class Kernel
+{
+    protected string $environment;
```

Comment:
CI wasn't triggered somehow and tests need to be run but this won't work with < PHP 7.4. `string` type can be added as a PHPDoc.